development/gcp/pkg/pubsub/client.go

```diff
@@ -1,15 +1,78 @@
 package pubsub
 
 import (
-	"cloud.google.com/go/pubsub"
 	"context"
 	"encoding/json"
+	"flag"
 	"fmt"
-	"github.com/google/go-github/v40/github"
 	"net/url"
 	"path"
+
+	"cloud.google.com/go/pubsub"
+	"github.com/google/go-github/v40/github"
```

Comment:
Use consistent GitHub client. Don't use `v40` when you are already using `v42`.

Reply:
Also, what `k8s.io/test-infra/` GitHub client is missing? Can't we just use that one instead?